sdk/dotnet/Pulumi.Automation/StackDeployment.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2016-2021, Pulumi Corporation
+
+using System;
+using System.Text.Json;
+
+namespace Pulumi.Automation
+{
+    /// <summary>
+    ///
+    ///  Represents the state of a stack depoyment ExportStackAsync
+    ///  and ImportStackAsync.
+    ///
+    /// </summary>
+    public sealed class StackDeployment
+    {
+        internal static StackDeployment FromJsonString(string jsonString)
+        {
+            var json = JsonSerializer.Deserialize<JsonElement>(jsonString);
+            var version = json.GetProperty("version").GetInt32();
+            return new StackDeployment(version, json, jsonString);
+        }
+
+        /// <summary>
+        /// Version indicates the schema of the encoded deployment.
+        /// </summary>
+        public int Version { get; }
+
+        /// <summary>
+        /// JSON repsresentation of the deployment.
+        /// </summary>
+        public JsonElement Json { get; }
```

Comment:
There's a typo in the comment here. Also `JsonElement` has [GetRawText](https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonelement.getrawtext?view=net-5.0#System_Text_Json_JsonElement_GetRawText) which gives you the raw JSON back